packages/ag-grid-community/src/interfaces/iAdvancedFilterBuilderParams.ts

```diff
@@ -27,4 +29,21 @@ export interface IAdvancedFilterBuilderParams {
      * @default 200
      */
     pillSelectMaxWidth?: number;
+    /**
+     * Specifies the buttons to be shown in the Advanced Filter Builder, in the order they should be displayed in.
+     * The options are:
+     *
+     *  - `'apply'`: The Apply button will apply the filter and close the builder.
+     *  - `'clear'`: The Clear button will clear the filter in the builder without removing the current active filter.
+     *  - `'reset'`: The Reset button will clear the filter and apply an empty filter.
+     *  - `'cancel'`: The Cancel button will discard any changes that have been made to the filter in the UI, restoring the applied model.
+     *
+     * @default ['apply', 'cancel']
+     */
+    buttons?: FilterAction[];
+    /**
+     * Whether to hide the Full Screen button in the Advanced Filter Builder:
+     * @default false
+     */
+    hideFullScreenButton?: boolean;
```

Comment:
```suggestion
    suppressFullScreenButton?: boolean;
```
We tend to use `suppress` in options